content/2022/audio-lokalizacija.md

```diff
@@ -23,6 +23,41 @@ Uvod treba da sadrži sledeće stvari:
 
 ### Aparatura i metoda
 #### Metod
+##### Koraci <!---mora neki bolji naziv i mozda da se smanje svi sledeci naslovi za jedan stepen--->
```

Comment:
Mislim da nema smisla samo da navedete ove korake kao listu, a da ih posle opet navedete sa objasnjenjem u **Izračunavanja distance**. Ja bih ovo pretvorio u dijagram na pocetku ovog odeljka, kome prethode par _high-level overview_ recenica u uvodu o tome koji sve metodi postoje (u literaturi) i zasto ste se bas odlucili za ove.

Onda u odeljku **Metod** opisete detaljno sve korake i referencirate se na taj dijagram. Nakon sve algoritmike ide aparatura (moze biti pododeljak metoda) u kojoj opisujete sta ste koristili od hardvera, koja su mu ogranicenja i tako to...